config/vufind/geofeatures.ini

```diff
@@ -1,8 +1,15 @@
 ; Geographic feature configuration settings
 ;
-; Currently all but the basemap configurations for the geographic feature
-; functionality are located in config.ini (for Map Tab display) and in
-; searches.ini (for Map Selection).
+; Legacy geographic feature configurations are located in
+; config.ini (for Map Tab display) and in searches.ini (for Map Selection).
+; Configuration settings in this file overwrite settings in the
+; config.ini and searches.ini files.
+
+[GeoPlatform]
+; Configures the mapping platform that should be used.
+; Replaces config.ini recordMap setting.
+; Options:  leaflet (default)
+geoPlatform = leaflet
```

Comment:
Do you think it is worth maintaining a geoPlatform setting if we're not worrying about backward compatibility here? It might simplify some things to rip it out. I don't mind keeping it if you think it has value, but if it's unlikely that we're going to offer alternate platforms, it might be just as well to get rid of it, and deal with reimplementing it in the future if the situation changes.

Reply:
If we were going to maintain two mapping platforms, then yes, but I'd rather keep this streamlined. I'll need to do some rewriting of the code per this suggestion and the ones above and below, but I think that will be good.